src/service/ConfigService.ts

```diff
@@ -349,15 +349,15 @@ export class ConfigService {
             addresses,
             serverConfig,
             NodeType.PEER_NODE,
-            (nodePresetData) => nodePresetData.harvesting,
+            (nodePresetData) => nodePresetData.harvesting && nodePresetData != nodePreset,
```

Comment:
sorry, I should have explained this better in the bug.
Each node should have new preset indicating whether its a static or dynamic node.
mainnet nodes are dynamic except the core nodes which are static.
Private bootstrap you can leave those as static.

Reply:
If you are creating a new node, not an ngl node, should your node be on your list?

And in terms of bootstrap private, each node only has the other ones in this change. Is that ok? Or should each node have all the nodes, including themselves in the peer lists